src/explorer.ts

```diff
@@ -300,6 +323,10 @@ export class OpenShiftExplorer implements TreeDataProvider<ExplorerItem>, Dispos
                     result = [...helmRepos.sort(Helm.ascRepoName)];
                 }
             }
+        } else if ('kind' in element && element.kind === 'Deployment') {
+            const pods = await Oc.Instance.getKubernetesObjects('pods');
+            const filteredPods: DeploymentPodObject[] = pods.filter((pod) => pod.metadata.name.indexOf(element.metadata.name) !== -1 && pod);
+            return filteredPods.filter((pod) => pod.status.phase === 'Running');
```

Comment:
May I ask what's the reason for only listing pods that are running? I think it could be helpful to show pods that are starting or stopping.

Reply:
> May I ask what's the reason for only listing pods that are running? I think it could be helpful to show pods that are starting or stopping.

+1

Reply:
Now listing all the `pods` from cluster